### ISSUE TITLE
Different sims obtained when using different number of MPI nodes

### ISSUE THREAD

**Issue by andreufont** [damonge/CoLoRe#28]
I get different simulations when I use the same seed, but different number of MPI nodes.

Is this a bug, or a well known feature? 

I found this while investigating issue #25 on quasars with Dec=NaN, since I got different number of bad Dec when using MPI and not using it.

**Comment by damonge** [comment]
@andreufont Yes, this is to be expected. The seeding happens differently depending on the number of nodes. Is this a problem?

**Comment by andreufont** [comment]
@damonge - no, not really. In general is nice when you get the same answer regardless of OpenMP / MPI setting, but as long as it is documented, this should not be an issue. 
I just though it was a potential bug that could be related to some of the other issues, but we can close this issue.

**Comment by damonge** [comment]
OK, thanks. I'll leave all issues open for the time being until we find the reason for the NaNs.

**Comment by andreufont** [comment]
Should we also close this one?

**Comment by damonge** [comment]
Let's leave it open in case someone else encounters this "issue"

**Comment by cramirezpe** [comment]
@Laura-Casas, thanks for reminding this... I was aware but at some point, I forgot it.
